unidist/core/backends/mpi/core/common.py

```diff
@@ -491,3 +492,30 @@ def is_shared_memory_supported():
         return False
 
     return True
+
+
+def is_runned_by_mpiexec():
+    """
+    Check if the unidist was running using `mpiexec` command.
+
+    Returns
+    -------
+    bool
+        True or False.
+    """
+    if "Intel" in MPI.Get_library_version() and os.getenv("PMI_RANK") is None:
+        return False
+
+    if "MPICH" in MPI.Get_library_version() and os.getenv("PMI_RANK") is None:
+        return False
```

Comment:
Can be combined.

Reply:
Yes, but why do it? I think it's more readable

Reply:
We do not need to add the same checks in different conditions. Upon combining, black will make the code clear.

Reply:
Yes, in this reason all checks, required for the shared memory, is located in the shared storage.